### PR TITLE
gui config (#326)

### PR DESCRIPTION
**Pull request by dinhlongviolin1**
Solve Issue #326 
- Move taipy into src folder
- fix tests by modyfing sys.modules
- fix additonal codeql path vulnerabilities

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "flask-talisman>=1.0,<2.0",
     "gevent>=21.12.0,<22.0",
     "gevent-websocket>=0.10.1,<0.11",
+    "taipy-config@git+https://git@github.com/Avaiga/taipy-config.git@develop",
```
Comment (FredLL-Avaiga) [inline setup.py:40]: ?

**Review by FredLL-Avaiga — APPROVED** [review]
get taipy-config from git@develop ?

File: setup.py (hunk 1 above)
Comment (dinhlongviolin1) [reply setup.py:40]: You are right. We are getting taipy-config from github (same for taipy-core). We will change this as soon as taipy-config is pushed to pypi

**Review by FabienLelaquais — APPROVED** [review]
Looking really good.
Let me update the doc build process to reflect that...

**Pull request merged**